packages/marshal/src/passStyleOf.js

```diff
@@ -362,9 +368,15 @@ function checkRemotable(val, check = x => x) {
   const p = getPrototypeOf(val);
 
   if (ALLOW_IMPLICIT_REMOTABLES && (p === null || p === objectPrototype)) {
+    const err = assert.error(
+      X`Remotables should be explicitly declared: ${q(val)}`,
+    );
+    console.warn('Missing Far:', err);
+    // eslint-disable-next-line no-debugger
+    debugger;
```

Comment:
I don't know the exact circumstances: will this turn on the deoptimiser?

Reply:
Sorry, wasn't meant for production. Gone.